packages/block-template-custom-element/package.json

```diff
@@ -24,6 +24,7 @@
     "format": "prettier --write --ignore-unknown .",
     "lint:eslint": "eslint .",
     "lint:tsc": "tsc --noEmit",
+    "prepublishOnly": "PACKAGE_DIR=$(pwd) yarn workspace @local/package-chores exe scripts/cleanup-before-publishing.ts",
```

Comment:
I assume that we won't need to publish from Windows. Running `yarn workspace @local/package-chores exe scripts/cleanup-before-publishing.ts` without `PACKAGE_DIR` gives us `packages/@local/package-chores` as `process.cwd`. This value is set by `yarn workspace`.